ui/src/app/tasks/task-schedules-destroy/task-schedules-destroy.component.html

```diff
@@ -0,0 +1,48 @@
+<div *ngIf="taskSchedules">
+  <div class="modal-header">
+    <h4 class="modal-title pull-left">Confirm Delete Task Schedules</h4>
```

Comment:
Should rename. Not exactly sure to what - Maybe (too wordy??): "Confirm the Deletion of Task Schedules"

![image](https://user-images.githubusercontent.com/685363/42483907-231943ea-838c-11e8-8953-78a8cdd8c7e8.png)

Reply:
@ghillert We chose 'Delete a schedule'